css-scroll-anchoring-1/Overview.bs

```diff
@@ -35,6 +35,20 @@ spec: html; urlPrefix: https://html.spec.whatwg.org/multipage/;
         type: dfn;
             text:DOM anchor
             text:focused area of the document
+            text:active match; url: fip-active-match
+    urlPrefix: form-elements.html
+        type: dfn;
+            text:textarea; url: the-textarea-element
+    urlPrefix: input.html
+        type: dfn;
+            text:input; url: the-input-element
+    urlPrefix: form-control-infrastructure.html
+        type: dfn;
+            text:mutable; url: concept-fe-mutable
```

Comment:
You should be able to get rid of these if you use `<{input}>` and `<{textarea}>` instead of `[=input=]` and `[=textarea=]`.

Reply:
Done. Thanks!